FAQ.md

```diff
@@ -586,52 +655,55 @@ As distributed transactions need to conduct two-phase commit and the bottom laye
-+------------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+#### Does TiDB support the `replace into` syntax?
 
-```
+Yes. But the `load data` does not support the `replace into` syntax.
 
-- The `OWNER` represents the TiDB server that is running this DDL statement.
-- The `JOB` lists the detailed information of the task.
-- The `SchemaID:81, TableID:90` in `JOB` represents the database ID and the user table ID.
-- The `RowCount:1293344122` in `JOB` represents the numer of rows that have been processed currently.
+#### How long does it take to reclaim disk space after deleting data?
 
-#### The `column Show_db_priv not found` message is displayed when executing `grant SHOW DATABASES on db.*`.
+None of the `Delete`, `Truncate` and `Drop` operations releases data immediately. For the `Truncate` and `Drop` operations, after the TiDB GC (Garbage Collection) time (10 minutes by default), the data is deleted and the space is released. For the `Delete` operation, the data is deleted but the space is not released according to TiDB GC. When data is written into RocksDB and executes `Compact`, the space is reused.
 
-`SHOW DATABASES` is a global privilege rather than a database-level privilege. Therefore, you cannot grant this privilege to a database. You need to grant all databases:
+#### Why does the query speed getting slow after deleting data?
 
-```
-grant SHOW DATABASES on *.*
-```
+Deleting a large amount of data leaves a lot of useless keys, affecting the query efficiency. Currently the Region Merge feature is in development, which is expected to solve this problem. For details, see the [deleting data section in TiDB Best Practices](https://pingcap.com/blog/2017-07-24-tidbbestpractice/#write).
 
-### SQL optimization
+#### What is the most efficient way of deleting data?
 
-#### How to view the query execution plan of TiDB?
+When deleting a large amount of data, it is recommended to use `Delete * from t where xx limit 5000;`. It deletes through the loop and uses `Affected Rows == 0` as a condition to end the loop, so as not to exceed the limit of transaction size. With the prerequisite of meeting business filtering logic, it is recommended to add a strong filter index column or directly use the primary key to select the range, such as `id >= 5000*n+m and id < 5000*(n+1)+m`.
+
+If the amount of data that needs to be deleted at a time is very large, this loop method will get slower and slower because each deletion traverses backward. After deleting the previous data, lots of deleted flags will remain for a short period (then all will be Garbage Collected) and influence the following Delete statement. If possible, it is recommended to refine the Where condition. See [details in TiDB Best Practices](https://pingcap.com/blog/2017-07-24-tidbbestpractice/#write).
```

Comment:
(then all will be Garbage Collected) → (then all will be dealt with by Garbage Collection)